mesop/server/config.py

```diff
@@ -11,9 +11,13 @@
 class Config(BaseModel):
   """Mesop app configuration."""
 
-  state_session_backend: Literal["memory", "file", "firestore", "none"] = "none"
+  state_session_backend: Literal[
+    "memory", "file", "firestore", "sql", "none"
+  ] = "none"
```

Comment:
I meant to ask this earlier; it's not clear why you need to set default values for these attributes since it looks like you're always setting a value for each attribute in CreateConfigFromEnv

Reply:
Yeah, I think initially I was considering the case where someone didn't use the factory function, but since internally the only usage is from the Factory, I've updated to set the defaults from factory function under the assumption that the config class generally wouldn't be instantiated without a factory function.

The other consideration was updating the factory function to only pass in the env vars that were set, thus allowing the defaults to be set by the config class.

Reply:
Actually, changed my mind. Switched to setting defaults in the config class. Then specifying parameter in the factory function only if the env var was set. Forgot I had some tests state session that were depending on this behavior.